randovania/data/json_data/prime3.txt

```diff
@@ -3763,24 +4616,37 @@ Landing Site Bravo
 Asset id: 4408585131827753141
 > Dock to Lift Hub Access; Heals? False
   * Normal Door to Lift Hub Access/Dock to Landing Site Bravo
+  > Samus Ship
+      Trivial
+
+> Samus Ship; Heals? True
+  > Dock to Lift Hub Access
+      Trivial
 
 ----------------
 Security Air Lock
 Asset id: 5682718801820042955
 > Dock to Lift Hub; Heals? False
   * Normal Door to Lift Hub/Dock to Security Air Lock
   > Dock to Defense Access
-      X-Ray Visor
+      After Security Air Lock Gate
+  > Pickup (Missile Expansion)
+      Trivial
```

Comment:
Remove this node

Reply:
The missile tank should already connect to lift hub and vice versa, and nothing else

Reply:
Ah never mind, I see why.